src/Tests/AuthoringTest/Program.cs

```diff
@@ -1854,6 +1854,17 @@ private static int GetNumberFromAbi(void* thisPtr, int* value)
             }
         }
     }
+
+    [System.Runtime.InteropServices.Guid("26D8EE57-8B1B-46F4-A4F9-8C6DEEEAF53A")]
+    public interface ICustomInterfaceGuid
+    {
+        string HelloWorld();
+    }
+
+    public sealed class CustomInterfaceGuidClass : ICustomInterfaceGuid

```

Comment:
Can you add this class to [here](https://github.com/microsoft/CsWinRT/blob/user/sergiopedri/custom-guid/src/Tests/AuthoringConsumptionTest/AuthoringConsumptionTest.exe.manifest).

Reply:
Done